Pollo/Views/Cards/MCResultCell.swift

```diff
@@ -148,8 +152,11 @@ class MCResultCell: UICollectionViewCell {
         }
 
         highlightView.snp.makeConstraints { make in
-            make.leading.top.bottom.trailing.equalToSuperview()
+            make.leading.top.bottom.equalToSuperview()
+            let highlightViewMaxWidth = Float(contentView.bounds.width - horizontalPadding * 2)
```

Comment:
I don't think you need `highlightViewMaxWidth`, because the max with will be the width of `containerView` (superView), right? Does it work if you just do `make.width.equalToSuperview().multipliedBy(percentSelected)`?